packages/office-ui-fabric-react/src/components/Rating/Rating.types.ts

```diff
@@ -26,6 +26,7 @@ export interface IRatingProps extends React.AllHTMLAttributes<HTMLElement> {
 
   /**
    * Minimum rating, defaults to 1, has to be >= 0
```

Comment:
It might be worthwhile to update the comment here. I'm not sure it was correct in the first place. 

`Deprecated: this prop has caused confusion and bugs, and is now ignored.`